.github/workflows/scripts_checking.yml

```diff
@@ -0,0 +1,30 @@
+name: Scripts quality CI
```

Comment:
The name does not reflect really the content, because it checks the source code also... Why not keeping the same name like in `app-boilerplate`: `Misspellings checks`?

Same for the yml script name: `misspellings_checks.yml`?

Reply:
Done